### DISCUSSION TITLE
Welcome to msgraph-sdk-go Discussions!

### DISCUSSION THREAD

**Discussion by baywet** [microsoftgraph/msgraph-sdk-go#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Please keep using issues when reporting bugs or asking for new features when you have enough details. Things can always start as a discussions if you don't feel like you have enough details right away and want to exchange first, we can always convert discussions into issues later on.

We're the team behind the msgraph-sdk-go, msgraph-beta-sdk-go and msgraph-sdk-go-core repositories:

- @maisarissi Program Manager
- @baywet Software developer

**Comment by kamalbana** [comment]
Hello Team,

I have updated msgraph-sdk-go from v0.6.0 -> v.0.22.0. which has broken some of our existing changes. 

For instance - 

result, err := graphClient.ApplicationsById(&applicationId).Get(nil)
used to return object of type - ApplicationsRequestBuilder but now returns ApplicationItemRequestBuilder. 

and i am not getting clear understanding on how to use this new kind of object. documentation still explains old version code.

https://docs.microsoft.com/en-us/graph/api/application-get?view=graph-rest-1.0&tabs=go

Can you please help me with some documentation or suggestions how to use this new kind of object

**Reply by kamalbana** [reply]
@baywet

**Reply by baywet** [reply]
@kamalbana can you created a [new thread](https://github.com/microsoftgraph/msgraph-sdk-go/discussions/new) for your question for clarity please? Once it's done, I'll delete that answer and lock this thread.
